st3/sublime_lib/pure_resource_path.py

```diff
@@ -48,13 +48,16 @@ def parts(self):
 
     @property
     def parent(self):
+        """The logical parent of the path:"""
```

Comment:
This docstring seems unfinished.

Reply:
I fixed the punctuation. Examples would probably be good, but not absolutely required.